src/Nordea/Components/DropdownFilter.elm

```diff
@@ -175,7 +183,10 @@ inputSearchView hasFocus searchString onInput onFocus =
                     [ Css.outline Css.none
                     , Css.border3 (Css.px 1) Css.solid Colors.blueNordea
                     ]
-                , Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                , if hasFocus then
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                  else
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 4) (Css.px 4)
                 , Css.boxShadow5 Css.inset (Css.px 0) (Css.px -1) (Css.px 0) Colors.grayLight
```

Comment:
Should this perhaps be removed? @TheaSommersletta 
![image](https://user-images.githubusercontent.com/1502013/146574608-fbb10271-591a-41c7-aec7-83caaad622e6.png)